pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -74,9 +75,12 @@ def get_check_function(check_function_name: str) -> callable:
     """
     Function used to fetch an arbitrary NWB Inspector function
     """
-    check_function: callable = nwbinspector.__dict__.get(check_function_name)
+    dandi_check_list = configure_checks(config=load_config(filepath_or_keyword="dandi"))
+    dandi_check_registry = {check.__name__: check for check in dandi_check_list}
+
+    check_function: callable = dandi_check_registry.get(check_function_name)
```

Comment:
```suggestion
    check_function: callable = dandi_check_registry[check_function_name]
```

If you swap this from `get` you can remove the `if .. is None; raise ...` below, since they do the same thing